src/index.html

```diff
@@ -1,19 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>To Do List</title>
-    <script src="https://kit.fontawesome.com/05c5795d1a.js" crossorigin="anonymous"></script>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Pokedex</title>
+	<script src="https://kit.fontawesome.com/05c5795d1a.js" crossorigin="anonymous"></script>
 <script defer src="main.js"></script></head>
 <body>
-    <nav></nav>
-    <section id="homePage">
-
-    </section>
-    <section id="popUp">
-        
-    </section>
-    <footer></footer>
+	<nav></nav>
+	<section id="homePage">
+			<div class="card">
+					<img class="pokemon-img" src="/src/images/charmander.jpeg" alt="charmander">
+					<div class="card-data">
+							<h2 class="pokemon-name">Pokemon 1</h2>
+							<div class="like-section"><i class="fa-regular fa-heart heart"></i><p>5 likes</p></div>
+					</div>
+					<button class="comment-button">Comments</button>
+			</div>
+			<div class="card">
+					<img class="pokemon-img" src="./images/charmander.jpeg" alt="charmander">
+					<div class="card-data">
+						<h2 class="pokemon-name">Pokemon 1</h2>
+						<div class="like-section"><i class="fa-regular fa-heart heart"></i><p>5 likes</p></div>
+				</div>
+					<button class="comment-button">Comments</button>
+			</div>
+			<div class="card">
+					<img class="pokemon-img" src="./images/charmander.jpeg" alt="charmander">
+					<div class="card-data">
+						<h2 class="pokemon-name">Pokemon 1</h2>
+						<div class="like-section"><i class="fa-regular fa-heart heart"></i><p>5 likes</p></div>
+				</div>
+					<button class="comment-button">Comments</button>
+			</div>
+			<div class="card">
+					<img class="pokemon-img" src="./images/charmander.jpeg" alt="charmander">
+					<div class="card-data">
+						<h2 class="pokemon-name">Pokemon 1</h2>
+						<div class="like-section"><i class="fa-regular fa-heart heart"></i><p>5 likes</p></div>
+				</div>
+					<button class="comment-button">Comments</button>
+			</div>
+			<div class="card">
+					<img class="pokemon-img" src="./images/charmander.jpeg" alt="charmander">
+					<div class="card-data">
+						<h2 class="pokemon-name">Pokemon 1</h2>
+						<div class="like-section"><i class="fa-regular fa-heart heart"></i><p>5 likes</p></div>
+				</div>
+					<button class="comment-button">Comments</button>
+			</div>
+			<div class="card">
+					<img class="pokemon-img" src="./images/charmander.jpeg" alt="charmander">
+					<div class="card-data">
+						<h2 class="pokemon-name">Pokemon 1</h2>
+						<div class="like-section"><i class="fa-regular fa-heart heart"></i><p>5 likes</p></div>
+				</div>
+					<button class="comment-button">Comments</button>
+			</div>
+	</section>
+	<section id="popUp">
```

Comment:
[OPTIONAL]  Please kindly change the `section` id to lowercase to match the naming conventions for CSS.

Reply:
Thank you Camilo for your comment. I will commit the changes and merge them to the dev branch.